CenterForOpenScience/ember-osf-web
# [HOLD][ENG-1686] Allow read and write permission users to see drafts tab

- Ticket: [ENG-1686]
- Feature flag: n/a

## Purpose
- Currently only admins can see draft registrations. Read- and Write-contributors should be able to see the draft registrations
- Read- and Write-contributors should not be able to start registrations

## Summary of Changes
- Updated condition that only allowed admins to see draft registrations to read-contributors

## Side Effects
- there should be none

## QA Notes
- As stated above, read- and write-contributors should be able to view the draft registrations. Further work will be needed to ensure that when these users go into draft registrations they have proper permissions to edit (Write: can edit fields but can't submit, Read: can only view fieds, can't edit or submit)


[ENG-1686]: https://openscience.atlassian.net/browse/ENG-1686
